src/pages/trade/HorizontalPanel.tsx

```
@@ -382,16 +382,24 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen, handleStartResize }:
   );
 
   const tabItems = useMemo(
-    () => [positionTabItem, ordersTabItem, fillsTabItem, orderHistoryTabItem, paymentsTabItem],
-    [positionTabItem, fillsTabItem, ordersTabItem, orderHistoryTabItem, paymentsTabItem]
+    () => [positionTabItem,
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)